src/lib/lambda.js

```diff
@@ -79,8 +80,39 @@ const getFunctionsInRegion = async region => {
 	return loop();
 };
 
+const getAllLambdasCount = async AWS => {
```

Comment:
can you rename to `getAllFunctionsCount` to be consistent with the rest of the module?

Reply:
also, can you rename the variables so they refer to functions as `functions` as opposed to `lambdas`, so the naming is consistent within the module and with the AWS SDK as well (`lambda` is the name of service, `function` is the name of the individual resources)